src/test/files/FreeResourcesOfAutoCloseableInterface2.java

```diff
@@ -0,0 +1,45 @@
+package files;
```

Comment:
Hi @rrousset-michelin,
thank you for the work !
do you think about adding this test case inside de "ecocode-java-test-project" to check in a real world if the problem is ok with a local SonarQube ?